packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -1,21 +1,14 @@
 import { CrossmintWalletService } from "@/api";
 import { EVMAAWallet, TChain, entryPoint, getBundlerRPC } from "@/blockchain";
-import type { BackwardsCompatibleChains, CrossmintAASDKInitParams, WalletConfig } from "@/types";
-import {
-    CURRENT_VERSION,
-    WalletSdkError,
-    ZERO_DEV_TYPE,
```

Comment:
we will need `ZERO_DEV_TYPE ` and `CURRENT_VERSION` for `storeAbstractWallet`, but we'll add them later